lessons/04-structured-output/README.md

```diff
@@ -0,0 +1,590 @@
+# Structured output
+
+In this chapter, you will learn the following:
+
+- **Structured output**, what it is and how to leverage it effectively. 
+- **Extract information from prompts** and reflect it on the output. 
```

Comment:
The wording needs fixed here. Maybe the following?

- **Extract key information from prompts** and incorporate it into the output.